examples/strategy_example.py

```
@@ -0,0 +1,65 @@
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+class MyRSIStrategy(PyStrategy):
+    def __init__(self, rsi_period=14):
+        super().__init__()
+        self.rsi_period = rsi_period
+        self.prices = {}
+
+    def on_start(self, ctx):
```

Ruff (0.14.14)

[warning] Unused method argument: `ctx`

(ARG002)